HISTORY.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.1.0: 2015-09-10
+
+- Updated dependencies
+- Switched to `extend-shallow`
```

Comment:
It would be good to have more of a description of the impact, rather than the internal change in the change log.  Perhaps something like switched to only doing a shallow merge of options and locals.